tests/fixtures/files.py

```diff
@@ -1,76 +1,82 @@
-import os
 from pathlib import Path
-from typing import TextIO
 
 import pytest
 
+from palace.manager.sqlalchemy.model.resource import Representation
+from tests.fixtures.database import DatabaseTransactionFixture
 
-class ResourcesFilesFixture:
-    """A fixture providing access to resource files."""
 
-    """These are files that are served by the CM in production, and aren't test suite specific."""
+class FilesFixture:
```

Comment:
This fixture is updated for the new import paths, and a number of the other separate files fixtures have all been centralized in this file.